troposphere/static/js/components/projects/detail/resources/InstanceActionButtons.jsx

```diff
@@ -59,6 +63,14 @@ export default React.createClass({
                 );
                 linksArray.push(
                     <Button style={style}
+                        key="Resize"
+                        icon="repeat"
```

Comment:
This icon should be `resize-full` as well for consistency. 

I believe this is for the Button Group above the Project Resources List view and is shown when an instance is checked. Note that what buttons are shown is controlled by what resources are selected, the state of that resource, and the cross section of actions available to ALL of the resources selected. For example a selected unattached Volume might have the actions delete and attach, but when an Instance is also selected only delete might show. 

I don't know off hand what exact actions are available under what specific state but that is the general idea of it.

This is something that @lenards has been working on so he can help if you're not seeing your Button when a healthy instance is selected.